tests/utils.py

```diff
@@ -76,3 +76,20 @@ def __init__(self):
 
     def __add__(self, x):
         return self.url + x
+
+
+def open_legacy_httpbin(browser, httpbin):
+    """Opens the start page of httpbin (given as a fixture). Tries the
+    legacy page (available only on recent versions of httpbin), and if
+    it fails fall back to the main page (which if JavaScript-only in
+    recent versions of httpbin hence usable for us only on old
```

Comment:
Minor typos:
* which _is_ JavaScript-only
* missing parenthesis at the end of the sentence.

Reply:
Fixed, thanks.